packages/core/src/lib/helpers/getActiveAccount.ts

```diff
@@ -0,0 +1,8 @@
+import { AccountState } from "../store.types";
+import { WalletSelectorStore } from "../wallet-selector.types";
+
+export const getActiveAccount = (
+  store: WalletSelectorStore
+): AccountState | null => {
+  return store.getState().accounts.find((account) => account.active) || null;
+};
```

Comment:
Maybe we could make this reusable in the dApp by switching this to use `WalletSelectorState` instead of the `WalletSelectorStore`